src/simulate/main.py

```diff
@@ -60,28 +61,52 @@ def main():
             sys.exit(1)
 
     files = os.listdir(json_files_dir)
+    session = requests.session()
     logging.info('list of json files in the directory %s', files)
+    event_count = 0
     for i in range(run_count):
         for file in files:
```

Comment:
Combine the `for` and the `if` it contains:

`for file in [file in files if file.endswith('.json')]:`

Reply:
I have simplified it now using `filter and lambda`